web/src/utils/datetime.ts

```diff
@@ -42,6 +42,9 @@ export const getTimeRangeOptions = (t: TFunction) => {
 };
 
 export const getFormattedDate = (date: Date, format = 'llll') => {
+  if (window.navigator.language.length && moment.locale() !== window.navigator.language) {
+    moment.locale(window.navigator.language);
+  }
```

Comment:
What's the reason for doing this?  On my machine, moment.locale() is "en" and window.navigator.language is "en-US".  Therefore, it will call moment.locale('en-US'), but that ends up getting changed to "en", and so this will get set on every call.

Also, you can drop "window." and just use navigator.language since it's implied.

Reply:
Moved to `navigator.language` + Intl.DateTimeFormat